From: Matthias Clasen <mclasen@redhat.com>
Date: Thu, 1 Aug 2024 14:43:49 -0400
Subject: vulkan: Don't try srgb for the atlas

This makes the Emoji rendering come out wrong.

Fixes: #6887
(cherry picked from commit 67080dab2171029c46bcb5250fab1b1900b5fba2)

Origin: upstream, after 4.15.4
---
 gsk/gpu/gskvulkanimage.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gsk/gpu/gskvulkanimage.c b/gsk/gpu/gskvulkanimage.c
index dc34cee..38b971d 100644
--- a/gsk/gpu/gskvulkanimage.c
+++ b/gsk/gpu/gskvulkanimage.c
@@ -523,7 +523,7 @@ gsk_vulkan_image_new_for_atlas (GskVulkanDevice *device,
 
   self = gsk_vulkan_image_new (device,
                                GDK_MEMORY_DEFAULT,
-                               TRUE,
+                               FALSE,
                                GSK_GPU_IMAGE_FILTERABLE | GSK_GPU_IMAGE_RENDERABLE,
                                width,
                                height,
